Controller/Adminhtml/Importcategory/Save.php

```diff
@@ -160,11 +160,13 @@ public function execute()
                                         }
                                         $cateitem->setAttributeSetId($cateitem->getDefaultAttributeSetId());
                                         $cateitem->setName($newcategory);
-                                        $_url_key = str_replace(' ', '-', strtolower($newcategory));
+
+                                        $_url_key =  $cat_data['url_key'] ? $cat_data['url_key'] : str_replace(' ', '-', strtolower($newcategory));
                                         if (in_array($newcategory, $exist_categories_name)) {
                                             $_url_key .= '-'.mt_rand(10, 99);
                                         }
                                         $cateitem->setUrlKey($_url_key);
+
```

Comment:
Is this new line necessary?

Reply:
why has this been resolved without a comment? Line is still there.